packages/generator/src/edmx-parser/legacy/common/edmx-to-vdm.ts

```diff
@@ -419,7 +422,7 @@ export function parseReturnType(
       isCollection: false
     };
   }
-  const isCollectionReturnType = isCollection(returnType);
+  const isCollectionReturnType = isCollectionType(returnType);
```

Comment:
Be consistent. I am a bit confused what `isCollectionType` is doing. I would assume it checks if a given type is a collection. So I would call it `isCollection(someType)` and would call all variables holding the return information `isCollection`.But now is se both: method is called `isCollectionType` and return value variables are sometimes called `isCollectionReturnType` or `isCollection`. So either use type or don't but do not mix.

Reply:
I changed this because we also have a function that is called `isComplexType` (where it makes a little bit more sense because it is called 'complex type' but anyways it is similar because we parse a type) and because this way in many places I can call the variable `isCollection` which I find more convenient.